### ISSUE TITLE
Button / FontAttributes

### ISSUE THREAD

**Issue by acaliaro** [michaeled/XenForms#1]
In FontAttributes I have 3 checkboxes so I can select None, Bold and Italic, but it's not useful to have None and the other 2 properties selectable at the same time

**Comment by michaeled** [comment]
Agreed. I'm a little undecided how to act on this, though. The [FontAttributes](https://developer.xamarin.com/api/type/Xamarin.Forms.FontAttributes/) enum can be used as flags, so this is builtin to Xamarin.Forms.
